### ISSUE TITLE
Running alm.tools on termux (android)

### ISSUE THREAD

**Issue by angelosc** [alm-tools/alm#404]
I'm trying to run alm.tools on  my android device using termux. I'am able to install it without any errors, but after starting it, I'm getting the following error:

fileListingWorker worker restarting. Don't know why it stopped with code: 1
CANNOT LINK EXECUTABLE: library "libcares.so" not found
projectServiceWorker worker restarting. Don't know why it stopped with code: 1
CANNOT LINK EXECUTABLE: library "libcares.so" not found

and the errors are repeated until I stop it (through ctrl-c)

Any ideas ?

**Comment by basarat** [comment]
Suspect chokidar or perhap fsevents doesn't work on termux. If that is indeed the case their install should have failed (better ux) instead of runtime. Typing on my mobile. Will give you steps to narrow down soon 🌹👌🏻

**Comment by basarat** [comment]
Use the following repro

```bash
mkdir foo 
cd foo
npm init -y
npm install chokidar --save
node -e "require('chokidar')";
```

Please report the error to https://github.com/paulmillr/chokidar/issues :rose: 

I personally don't have a `termux` setup :heart:

**Comment by dontsave** [comment]
I have been trying to get this great project to work in Termux on Android as well. I get the same worker issues described above, however chokidar doesn't seem to be the problem. 

`mkdir foo 
cd foo
npm init -y
npm install chokidar --save
node -e "require('chokidar')";`

runs fine. Also installing chokidar and running node chokidar-test.js with:

`var chokidar = require('chokidar');
chokidar.watch('.', {ignored: /[\/\\]\./}).on('all', (event, path) => {
  console.log(event, path);
});`

correctly executes and logs events to the console.

Any other idea why fileListingWorker and projectServiceWorker might be failing at runtime? It would be amazing to have alm running on an Android tablet. It's the perfect portable device for developing on the go. Right now I'm stuck with vim while traveling, but have been dying for a better editor like alm for some time now.

**Comment by basarat** [comment]
That moment when my code is the only one experiencing the problem

![image](https://cloud.githubusercontent.com/assets/874898/23090790/8a2f7156-f5fa-11e6-9197-94984b3071a0.png)

😄  😢  :rose:

**Comment by basarat** [comment]
@dontsave using `su`? (ref https://github.com/termux/termux-app/issues/112#issuecomment-230656521)

**Comment by dontsave** [comment]
@basarat not in su. the error happens as a normal user. su alm just results in alm command not found

**Comment by basarat** [comment]
@dontsave released with `2.33.0` give it a go! :rose: 

/cc @notoriousb1t the attached commit https://github.com/alm-tools/alm/commit/eddbc6715d4d27f3436a0adc5bc4387db738a772 essentially breaks atom support. Some gory memories https://github.com/atom/atom/issues/2887#issuecomment-93887882 :rose:

**Comment by dontsave** [comment]
works great now!
